modules/recommended-etcd-practices.adoc

```diff
@@ -20,15 +20,14 @@ To achieve such performance, run etcd on machines that are backed by SSD or NVMe
 The load on etcd arises from static factors, such as the number of nodes and pods, and dynamic factors, including changes in endpoints due to pod autoscaling, pod restarts, job executions, and other workload-related events. To accurately size your etcd setup, you must analyze the specific requirements of your workload. Consider the number of nodes, pods, and other relevant factors that impact the load on etcd.
 ====
 
-The following hard disk features provide optimal etcd performance:
-
-* Low latency to support fast read operation.
-* High-bandwidth writes for faster compactions and defragmentation.
-* High-bandwidth reads for faster recovery from failures.
-* Solid state drives as a minimum selection, however NVMe drives are preferred.
-* Server-grade hardware from various manufacturers for increased reliability.
-* RAID 0 technology for increased performance.
-* Dedicated etcd drives. Do not place log files or other heavy workloads on etcd drives.
+The following hard drive practices provide optimal etcd performance:
+
+* Use dedicated etcd drives. Do not place log files or other heavy workloads on etcd drives.
```

Comment:
Can we add a clear statement specifying the etcd drives should avoid over-the-network disks (e.g. avoid iSCSI disks for this purpose). When using iSCSI disks the OS & etcd partitions should not be split across two disks.

Reply:
Should we mention that the partitions should not be split across disks on iSCSI if we're explicitly informing the reader to avoid iSCSI drives?

Reply:
true. That's good.